.github/workflows/build.yml

```diff
@@ -87,7 +87,10 @@ jobs:
       run: |
         # can't do a tool restore here due to an 8.0.100 rc2 bug - this is fixed in GA so can go back to
         # dotnet tool restore then.
-        dotnet tool install paket --version 8.0.0-alpha002 && dotnet tool install fantomas --version 6.2.2 && dotnet paket restore
+        dotnet tool install paket --version 8.0.0-alpha002 && dotnet tool install fantomas --version 6.2.3 && dotnet paket restore
```

Comment:
now that GA has released we can probably remove this in favor of the 'normal' tool restore. I can look into that on main though.